### ISSUE TITLE
The helmich/typo3-typoscript-parser 2.3.0 update causes complaints about empty lines missing their indentation

### ISSUE THREAD

**Issue by oliverklee** [martin-helmich/typo3-typoscript-lint#116]
Steps to reproduce:

```bash
git clone git@github.com:TYPO3-Documentation/tea.git
cd tea
composer require --dev helmich/typo3-typoscript-lint
.Build/vendor/bin/typoscript-lint -c Configuration/TsLint.yml -vvv Configuration/TypoScript
composer require --dev helmich/typo3-typoscript-parser:~2.2.1
.Build/vendor/bin/typoscript-lint -c Configuration/TsLint.yml -vvv Configuration/TypoScript
```

With `helmich/typo3-typoscript-parser` 2.2.1, TS lint does not complain about empty lines having no indentation at all. With `helmich/typo3-typoscript-parser` 2.3.0, TS lint expects the empty lines to have indentation as well.

I'm not sure whether this is a bug of `helmich/typo3-typoscript-parser` or `helmich/typo3-typoscript-lint`. So I'm starting here. Could you please move it if it's in the wrong project?

**Comment by martin-helmich** [comment]
Thanks for the report. If it's alright with you, I'll be closing it as dupliate of #115, though.

**Comment by oliverklee** [comment]
Thanks, @martin-helmich! :heart:
